app/src/pages/movies/filter/Filter.jsx

```diff
@@ -1,29 +1,39 @@
 import { FilterListComponent } from '@pages/movies/filter/components/layouts/FilterListComponent';
 import { FilterOption } from '@pages/movies/filter/components/FilterOption';
-import options from '@assets/data/genres.json'
+import optionsFromJson from '@assets/data/genres.json'
 import React from 'react';
 
 class Filter extends React.Component {    
     state = {
-        selectedOption: options[0]
+        options: []

```

Comment:
selectedOption: null 

should be here as well - use the same shape when set State

Reply:
Fixed